### ISSUE TITLE
Why is the backward for bn layer stopped after 60 epoch?

### ISSUE THREAD

**Issue by developer0hye** [yuhuixu1993/BNET#1]
https://github.com/yuhuixu1993/BNET/blob/5d0443c146ac1b66b52eee2491cdaad5c5e1fff6/classification/imagenet/train_imagenet.py#L217

Thanks for sharing your work!

Why is the backward for bn layer stopped after 60 epoch?

Is the way commonly used to improve performance?

**Comment by yuhuixu1993** [comment]
> https://github.com/yuhuixu1993/BNET/blob/5d0443c146ac1b66b52eee2491cdaad5c5e1fff6/classification/imagenet/train_imagenet.py#L217
> 
> Thanks for sharing your work!
> 
> Why is the backward for bn layer stopped after 60 epoch?
> 
> Is the way commonly used to improve performance?

Thank you for your interest of our work. We use it in the imagenet training to solve the overfit problem when the kernel size of BNET is as large as 5 or 7.  It may not improve the performance of BNET-3 but BNET-5 and BNET-7 will be improved.

**Comment by developer0hye** [comment]
@yuhuixu1993 
Thanks for your reply!

I am very interested in your work!

I want to design the lightweight backbone for object detection. I hope that this method helps to improve the performance!

**Comment by developer0hye** [comment]
@yuhuixu1993 
Do you consider overfit problem in the object detection task using this method?

**Comment by yuhuixu1993** [comment]
> @yuhuixu1993
> Do you consider overfit problem in the object detection task using this method?

When I used the pretrained model to finetune on ImageNet, I did not meet with the overfit problem. You can refer to the Loss image in my paper. And I did not try to train from scratch on COCO.

**Comment by developer0hye** [comment]
@yuhuixu1993 
Thanks for your reply!

**Comment by yuhuixu1993** [comment]
> @yuhuixu1993
> Thanks for your reply!
> 
> I am very interested in your work!
> 
> I want to design the lightweight backbone for object detection. I hope that this method helps to improve the performance!

I am very interested in your work! Any progress or problem please let me know!

**Comment by developer0hye** [comment]
@yuhuixu1993 
Yes! You are so kind.

I have a plan to use the BNET-ResNet18 as the backbone of CenterNet!

[It](https://github.com/developer0hye/Simple-CenterNet)'s my CenterNet-Implementation link.
